theme/default/index.js

```diff
@@ -115,6 +119,32 @@ window.addEventListener('DOMContentLoaded', () => {
         });
     }
 
+    let termsConditionsContainer = dom.getElement('#terms-conditions');
+    register.querySelector('#term_conditions').addEventListener('click', (ev) => {
+        ev.preventDefault();
+        termsConditionsContainer.classList.remove('hidden');
+        reldens.gameDom.getJSON(reldens.appServerUrl + '/terms-and-conditions', (err, response) => {
+            console.log(err);
+            console.log(response);
+            termsConditionsContainer.querySelector('.loading-container').style.display = 'none';
+            termsConditionsContainer.querySelector('.tacbox').style.display = 'block';
```

Comment:
not sure if node.querySelector was used in the rest of the file but try to use gameDom.getElement instead

Reply:
yep... it is used all over the file. I'm going to change it on my code.